### PR TITLE
FEATURE: BB-1387 Pivot table column attributes grouping

### PR DESCRIPTION
**Pull request by xMort**

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-opensource-repository-check/29/ : SUCCESS in 10s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/37/ : SUCCESS in 38s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-opensource-repository-check/30/ : SUCCESS in 11s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/38/ : SUCCESS in 56s

File: docs/pivot_table_component.md [hunk 1]
```diff
@@ -169,6 +169,18 @@ const sortBy = [
 </div>
 ```
 
+## Grouping
+
+By default, the same values in column attributes are grouped across the rows, which help improves data readability in pivot tables. The header of the grouped columns is displayed at the top of the table even when you scroll through a long group.
+
+The grouping is applied when the following conditions are both met:
+* The table is sorted by the **first** column attribute (both in ascending and descending order).
```
Comment (evgeniabogranskaya) [inline docs/pivot_table_component.md]: Is it really 'both asc and desc' or rather 'either asc or desc'? Thanks.

**Comment by evgeniabogranskaya** [comment]
Proofread, edits provided, 1 comment --> LGTM

**Comment by evgeniabogranskaya** [comment]
LGTM

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-opensource-repository-check/31/ : SUCCESS in 17s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/39/ : SUCCESS in 1m 13s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-opensource-repository-check/32/ : SUCCESS in 18s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/40/ : SUCCESS in 31s

**Comment by strouhaljan** [comment]
LGTM

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-merge-compliance-check/9/ : SUCCESS in 30s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/41/ : SUCCESS in 49s

**Pull request merged**